server/src/main/java/io/unitycatalog/server/persist/RepositoryFactory.java

```diff
@@ -0,0 +1,36 @@
+package io.unitycatalog.server.persist;
+
+import io.unitycatalog.server.persist.utils.FileOperations;
+import io.unitycatalog.server.utils.ServerProperties;
+import java.util.HashMap;
+import java.util.Map;
+import lombok.Getter;
+import org.hibernate.SessionFactory;
+
+public class RepositoryFactory {
```

Comment:
please add class docs explaining what this is

Reply:
nit: is this really a "factory pattern". typically when you invoke `factory.getX(typeOfX)`, each invocation returns a new object. This class is just returning the same object every time for the same time.

Reply:
maybe just call it Repositories?